projects/Flux/dev.Dockerfile

```diff
@@ -0,0 +1,43 @@
+FROM nvidia/cuda:10.1-cudnn7-devel
+# To use this Dockerfile:
+# 1. `nvidia-docker build -f dev.Dockerfile -t flux_detectron2:v0 ../..`
+# 2. `nvidia-docker run -d -v /home/joan/.aws/:/root/.aws/ -v /home/joan/cvdev:/root/cvdev -p 8889:8888 -p 6007:6006 -p 23:22 --name flux_detectron_joan_gpu0 flux_detectron2:v0`
+
+
+ENV DEBIAN_FRONTEND noninteractive
+
+RUN apt-get update && apt-get install -y \
+	libpng-dev libjpeg-dev python3-opencv ca-certificates \
+	python3-dev build-essential pkg-config git curl wget automake libtool && \
+  rm -rf /var/lib/apt/lists/*
+
+RUN curl -fSsL -O https://bootstrap.pypa.io/get-pip.py && \
+	python3 get-pip.py && \
+	rm get-pip.py
+
+# Install detectron2 pypi dependencies
+# See https://pytorch.org/ for other options if you use a different version of CUDA
+RUN pip install torch torchvision cython 'git+https://github.com/facebookresearch/fvcore'
```

Comment:
It probably doesn't matter too much, but I think the recommended way of installing PyTorch is via Conda.